global_segment_map_node/cfg/default.yaml

```diff
@@ -11,21 +12,22 @@ voxblox:
 
 pairwise_confidence_merging:
   enable_pairwise_confidence_merging: true
-  merging_min_overlap_ratio: 0.15
-  merging_min_frame_count: 1
+  merging_min_overlap_ratio: 0.1
+  merging_min_frame_count: 2
 
 object_database:
   publish_gsm_updates: true
-  max_segment_age: 5
+  max_segment_age: 10
   scene_gsm_update_topic: "gsm_updates"
```

Comment:
I just realized the default topic name in the Controller is gsm_update, can we stick to singular also here?